deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Net45.GAC.wxs

```diff
@@ -45,6 +45,11 @@
               Source="$(var.TracerHomeDirectory)\net45\System.Runtime.InteropServices.RuntimeInformation.dll"
               KeyPath="yes" Checksum="yes" Assembly=".net"/>
       </Component>
+      <Component Win64="$(var.Win64)">
```

Comment:
Quick question, did you generate this with the `PrepareRelease` tool? I want to make sure this will be generated automatically by that tool

Reply:
To put things into the managed profiler output automatically (which is what dictates what is in the MSI output), we have a somewhat hacky way to include assemblies that it doesn't directly depend on, like `Datadog.Trace.AspNet` as seen here: https://github.com/DataDog/dd-trace-dotnet/blob/92d3593adc1246a9b50bc16d434d19d9244db629/src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj#L40

You may need to do the same in order to include this in the output automatically.

Reply:
Mmm no, didn't know about that tool, tomorrow I'll check it. Thanks!

Reply:
I've added the project and ran the `PrepareRelease` in https://github.com/DataDog/dd-trace-dotnet/pull/811/commits/52145a24eddfb6c80459294ee06a2e9335becdd9